### ISSUE TITLE
`java.lang.UnsatisfiedLinkError` when launching example app on Android

### ISSUE THREAD

**Issue by flikkr** [mapbox/mapbox-maps-flutter#312]
When trying to launch the example app, I get the following crash on app launch.

```
FATAL EXCEPTION: main
                 Process: com.mapbox.maps.flutter.example, PID: 10250
                 java.lang.RuntimeException: Unable to get provider androidx.startup.InitializationProvider: androidx.startup.StartupException: androidx.startup.StartupException: androidx.startup.StartupException: java.lang.UnsatisfiedLinkError: dlopen failed: library "libmapbox-common.so" not found
                 	at android.app.ActivityThread.installProvider(ActivityThread.java:7427)
                 	at android.app.ActivityThread.installContentProviders(ActivityThread.java:6939)
                 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6710)
                 	at android.app.ActivityThread.access$1500(ActivityThread.java:247)
                 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2053)
                 	at android.os.Handler.dispatchMessage(Handler.java:106)
                 	at android.os.Looper.loopOnce(Looper.java:201)
                 	at android.os.Looper.loop(Looper.java:288)
                 	at android.app.ActivityThread.main(ActivityThread.java:7839)
                 	at java.lang.reflect.Method.invoke(Native Method)
                 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
                 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
                 Caused by: androidx.startup.StartupException: androidx.startup.StartupException: androidx.startup.StartupException: java.lang.UnsatisfiedLinkError: dlopen failed: library "libmapbox-common.so" not found
                 	at androidx.startup.AppInitializer.doInitialize(AppInitializer.java:162)
                 	at androidx.startup.AppInitializer.discoverAndInitialize(AppInitializer.java:198)
                 	at androidx.startup.InitializationProvider.onCreate(InitializationProvider.java:38)
                 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2404)
                 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2374)
                 	at android.app.ActivityThread.installProvider(ActivityThread.java:7422)
                 	at android.app.ActivityThread.installContentProviders(ActivityThread.java:6939) 
                 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6710) 
                 	at android.app.ActivityThread.access$1500(ActivityThread.java:247) 
                 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2053) 
                 	at android.os.Handler.dispatchMessage(Handler.java:106) 
                 	at android.os.Looper.loopOnce(Looper.java:201) 
                 	at android.os.Looper.loop(Looper.java:288) 
                 	at android.app.ActivityThread.main(ActivityThread.java:7839) 
                 	at java.lang.reflect.Method.invoke(Native Method) 
                 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548) 
                 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003) 
                 Caused by: androidx.startup.StartupException: androidx.startup.StartupException: java.lang.UnsatisfiedLinkError: dlopen failed: library "libmapbox-common.so" not found
                 	at androidx.startup.AppInitializer.doInitialize(AppInitializer.java:162)
                 	at androidx.startup.AppInitializer.doInitialize(AppInitializer.java:148)
                 	at androidx.startup.AppInitializer.discoverAndInitialize(AppInitializer.java:198) 
                 	at androidx.startup.InitializationProvider.onCreate(InitializationProvider.java:38) 
                 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2404) 
                 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2374) 
                 	at android.app.ActivityThread.installProvider(ActivityThread.java:7422) 
                 	at android.app.ActivityThread.installContentProviders(ActivityThread.java:6939) 
                 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6710) 
                 	at android.app.ActivityThread.access$1500(ActivityThread.java:247) 
                 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2053) 
                 	at android.os.Handler.dispatchMessage(Handler.java:106) 
                 	at android.os.Looper.loopOnce(Looper.java:201) 
                 	at android.os.Looper.loop(Looper.java:288) 
                 	at android.app.ActivityThread.main(ActivityThread.java:7839) 
                 	at java.lang.reflect.Method.invoke(Native Method) 
                 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548) 
                 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003) 
                 Caused by: androidx.startup.StartupException: java.lang.UnsatisfiedLinkError: dlopen failed: library "libmapbox-common.so" not found
                 	at androidx.startup.AppInitializer.doInitialize(AppInitializer.java:162)
                 	at androidx.startup.AppInitializer.doInitialize(AppInitializer.java:148)
                 	at androidx.startup.AppInitializer.doInitialize(AppInitializer.java:148) 
                 	at androidx.startup.AppInitializer.discoverAndInitialize(AppInitializer.java:198) 
                 	at androidx.startup.InitializationProvider.onCreate(InitializationProvider.java:38) 
                 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2404) 
                 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2374) 
                 	at android.app.ActivityThread.installProvider(ActivityThread.java:7422) 
                 	at android.app.ActivityThread.installContentProviders(ActivityThread.java:6939) 
                 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6710) 
                 	at android.app.ActivityThread.access$1500(ActivityThread.java:247) 
                 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2053) 
                 	at android.os.Handler.dispatchMessage(Handler.java:106) 
                 	at android.os.Looper.loopOnce(Looper.java:201) 
                 	at android.os.Looper.loop(Looper.java:288) 
                 	at android.app.ActivityThread.main(ActivityThread.java:7839) 
                 	at java.lang.reflect.Method.invoke(Native Method) 
                 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548) 
                 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003) 
                 Caused by: java.lang.UnsatisfiedLinkError: dlopen failed: library "libmapbox-common.so" not found
                 	at java.lang.Runtime.loadLibrary0(Runtime.java:1077)
                 	at java.lang.Runtime.loadLibrary0(Runtime.java:998)
                 	at java.lang.System.loadLibrary(System.java:1656)
                 	at com.mapbox.common.loader.MapboxLibraryLoader.load(MapboxLibraryLoader.kt:19)
                 	at com.mapbox.common.CoreInitializer.create(CoreInitializer.kt:69)
                 	at com.mapbox.common.CoreInitializer.create(CoreInitializer.kt:11)
                 	at androidx.startup.AppInitializer.doInitialize(AppInitializer.java:155)
                 	at androidx.startup.AppInitializer.doInitialize(AppInitializer.java:148) 
                 	at androidx.startup.AppInitializer.doInitialize(AppInitializer.java:148) 
                 	at androidx.startup.AppInitializer.discoverAndInitialize(AppInitializer.java:198) 
                 	at androidx.startup.InitializationProvider.onCreate(InitializationProvider.java:38) 
                 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2404) 
                 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2374) 
                 	at android.app.ActivityThread.installProvider(ActivityThread.java:7422) 
                 	at android.app.ActivityThread.installContentProviders(ActivityThread.java:6939) 
                 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6710) 
                 	at android.app.ActivityThread.access$1500(ActivityThread.java:247) 
                 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2053) 
                 	at android.os.Handler.dispatchMessage(Handler.java:106) 
                 	at android.os.Looper.loopOnce(Looper.java:201) 
                 	at android.os.Looper.loop(Looper.java:288) 
                 	at android.app.ActivityThread.main(ActivityThread.java:7839) 
                 	at java.lang.reflect.Method.invoke(Native Method) 
                 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548) 
                 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003) 
```

**Comment by baleboy** [comment]
Does this happen every time or only occasionally? I wonder if it could be related to https://github.com/mapbox/mapbox-maps-android/issues/1109

**Comment by evil159** [comment]
@flikkr this issue should be addressed in 1.0.0. I'm closing this issue, feel free to reopen if you have any feedback.
